### PR TITLE
Profile-image changes

### PR DESCRIPTION
**Pull request by Sandhu-Harsh**

### CODE REVIEW

File: lib/actions.ex [hunk 1]
```diff
@@ -228,7 +228,12 @@ defmodule ExTus.Actions do
   def post(conn, create_cb) do
     headers = Utils.read_headers(conn)
 
-    upload_type = headers["upload-type"]
+    # Identify path for Profile Image
+    upload_type = if conn.request_path == "/profile-image" do
```
Comment (rajatkooapp) [inline lib/actions.ex]: change here as well....as we will use `/profiles` as path

File: lib/config.ex [hunk 2]
```diff
@@ -2,6 +2,7 @@ defmodule ExTus.Config do
   # this is some hard code config, it will be configurable
   @upload_url  "/files"
   @video_upload_url "/videos"
+  @profile_image_url "/profile-image"
```
Comment (rajatkooapp) [inline lib/config.ex]: change here the path as well

File: test/extus_test.exs [hunk 3]
```diff
@@ -2,6 +2,14 @@ defmodule ExtusTest do
   use ExUnit.Case
   doctest Extus
 
+  def callback_func_for_post(file_info, headers) do
+    assert Map.get(file_info, :size) == 1919488
+    assert Enum.at(headers, 0) == {"upload-length", "1919488"}
+    assert Enum.at(headers, 1) == {"upload-metadata",
+             "filename L1VzZXJzL3JhamF0Y2hhdWRoYXJ5L3dvcmsvZ2l0aHViL2ZpbGVfdXBsb2FkL3Rlc3QvZmlsZXMvdG1wX3ZpZGVvLm1wNA=="}
+    assert Enum.at(headers, 2) == {"user_id", "37ba9201-0f97-4b71-a2d2-829624203445"}
```
Comment (rajatkooapp) [inline test/extus_test.exs:10]: good

File: test/extus_test.exs [hunk 4]
```diff
@@ -16,4 +24,67 @@ defmodule ExtusTest do
...
   end
+
+  # test for profile-image path in get_upload_location
+  test "get_upload_location profile image" do
+    identifier = "12333"
+    upload_type = "PROFILE_IMAGE"
+    conn = %Plug.Conn{
+    }
+    result = ExTus.Actions.get_upload_location(conn, upload_type, identifier)
+    expected_result = "https://www.example.com/profile-image/#{identifier}"
```
Comment (rajatkooapp) [inline test/extus_test.exs]: good

**Review by rajatkooapp — CHANGES_REQUESTED** [review]
this is good. Just change the paths

File: lib/config.ex (hunk 2 above)
Comment (rajatkooapp) [reply lib/config.ex]: change the variable name too

File: lib/actions.ex (hunk 1 above)
Comment (Sandhu-Harsh) [reply lib/actions.ex]: done

File: lib/config.ex (hunk 2 above)
Comment (Sandhu-Harsh) [reply lib/config.ex]: done

**Review by rajatkooapp — APPROVED** [review]

**Pull request merged**